### ISSUE TITLE
Hide posts that mentioned blocked users

### ISSUE THREAD

**Issue by JaneValleyhasBtaken** [mastodon/mastodon#24436]
### Pitch

Hide posts that whether mentioned or contain users that I blocked.
Since I bloke a user or users, I probably won't see them or interactions with them. Beacuse that just make me very frustrated, and I believe a lot of users have the same feeling. But current behaviour of mastodon doesn't hide it. So please implement it.

### Motivation

This will make the experience of using Mastodon much, much better, I believe.

**Comment by SoniEx2** [comment]
no, please don't put multiple functionality under the same functionality.

we think mutes do this. we don't think mutes *should* do this, ideally you'd use a keyword filter if you really wanted to, but eh anyway.
